src/ice-breaker/about.md

```diff
@@ -47,3 +47,17 @@ cargo run add-person $your_user_name
 Example PRs:
 
 * [Example of adding yourself to the LLVM ICE-breakers.](https://github.com/rust-lang/team/pull/140)
+
+## Tagging an issue for an ICE-breaker group
+
+To tag an issue as appropriate for an ICE-breaker group, you give
+[rustbot] a [`ping`] command with the name of the ICE-breakers
```

Comment:
Do [rustbot] and [`ping`] labels miss links?

Reply:
oops yes